airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/CdcPostgresSourceTest.java

```diff
@@ -84,11 +85,9 @@ protected void setBaseImage() {
 
   @Override
   protected void assertExpectedStateMessageCountMatches(final List<? extends AirbyteStateMessage> stateMessages, long totalCount) {
-    // Count has been disabled due to we do not support RFR for non resumeable full refresh.
-    // AtomicLong count = new AtomicLong(0L);
-    // stateMessages.stream().forEach(stateMessage ->
-    // count.addAndGet(stateMessage.getSourceStats().getRecordCount().longValue()));
-    // assertEquals(totalCount, count.get());
+    AtomicLong count = new AtomicLong(0L);
```

Comment:
Just a question for my curiosity: since we don't have any concurrent operations in our implementation, why do we need an atomic variable?

Reply:
likely not needed at all - as long as we can compare the count I think we should be good

Reply:
oh i remembered - lambda function doesn't like me defining a long count = 0 then count += stateMessage.getCount() because it needs to be final